.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Request a new format specification to be created'
+labels: spec-request
+---
+<!--
+A request for a format specification consist of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete the fields without data and all the comments from this template (including this one!).
+The resulting YAML blocks should contain valid YAML.
+-->
+```yaml
+# This block is the boilerplate of a KSY spec file. Must be a valid KSY. Meant to be ready to be copied as it is.
+meta:
+  id: # insert an id here. KS currently has no namespaces, so the id MUST be unique within all the repo and this property should be futureproof.
+  title: # insert format human-readable name here
+  application: # Insert the application human-readable name here
+  file-extension:
+    # a YAML list of file extensions here
+  encoding: UTF-8
+  endian: le
+  xref: # check if the following sources contain info about the format you request
+    mime: # MIME type
+    rfc: # Link to RFC
+    forensicswiki: # Page name on https://forensicswiki.org/
+    justsolve: # ID on http://fileformats.archiveteam.org/wiki/
+    pronom: # IDs on https://www.nationalarchives.gov.uk/pronom/fmt/
+     - x-fmt/ 
+    wikidata: Q # ID on WikiData
+    fileformat: # Link to https://www.fileformat.info
+    loc: # ID in https://www.loc.gov/preservation/digital/formats/
+    din:
+      id: # DIN standard identifier
+
+doc: |
+  # Describe your format briefly here AND links to the sources of sample files.
+
+doc-ref:
+  # a YAML array of URIs to information sources and/or their names. See https://doc.kaitai.io/ksy_reference.html#doc-ref
+  # An effective strategy may be to search specifically for PDFs (filetype:pdf in Google) and source code on GitHub (links to source code files are especially valuable for understanding poorly documented parts).
+  # Prefer sources with a public-domain-like licenses over permissive ones, permissive ones over copyleft ones and free licenses over proprietary ones
+```
+
+```yaml
+# This block carries some machine-readable and editable information. It is meant to be used by a bot.
+issues:
+  # add here the ids of issues in https://github.com/kaitai-io/kaitai_struct ...
+  block:
+    # ... required to be solved in order to implement this format
+  affect:
+    # ... not strictly required to be solved before this format implementation, but causing us to use temporary solutions
```

Comment:
I think these two comments should be reworded a little so that they can be read as standalone sentences. It's a little confusing to have two continuations of the same sentence.

* block: Issues that must be solved in order to implement this format
* affect: Issues that are not strictly required to be solved before this format implementation, but are causing us to use workarounds

Reply:
Your variant doesn't have an emphasis that only numbers are meant to be placed there. Putting whole URIs maynbe beneficial, but they are large, compating them by replacing repo names with their abbreviations ... meajs we will have to explain abbreviation expansion scheme...